openapi-diff/src/modeler/AutoRest.Swagger.Tests/Resource/Swagger/old/added_readonly_required_property.json

```diff
@@ -0,0 +1,49 @@
+﻿{
+  "swagger": 2.0,
+  "info": {
+    "title": "added_required_property",
+    "version": "1.0"
+  },
+  "host": "localhost:8000",
+  "schemes": [ "http", "https" ],
+  "consumes": [ "text/plain", "text/json" ],
+  "produces": [ "text/plain" ],
+  "paths": {
+    "/api/Parameters": {
+      "put": {
+        "tag": [ "Parameters" ],
+        "operationId": "Parameters_Put",
+        "produces": [
+          "text/plain"
+        ],
+        "parameters": [
+          {
+            "name": "database",
+            "in": "body",
+            "required": true,
+            "type": "object",
+            "schema": { "$ref": "#/definitions/Database" }
+          }
+        ]
+      }
+    }
+  },
+  "definitions": {
+    "Database": {
+      "properties": {
+        "a": {
+          "type": "string",
+          "description": "Enum.",
+          "enum": [ "a1", "a2", "a3" ]
+
+        },
+        "b": {
+          "type": "string",
+          "description": "Enum.",
+          "enum": [ "b1" ],
+          "readOnly": true
```

Comment:
FYI this `readOnly` here is the key difference. All the rest was copied over from `added_required_property.json`. Same for `new` variant.